radio/src/targets/common/arm/stm32/timers_driver.cpp

```diff
@@ -66,7 +66,7 @@ void interrupt1ms()
   ++pre_scale;
 
   // 1 ms loop
-#if not defined(SIMU) && (defined(RADIO_FAMILY_T16) || defined(PCBNV14))
```

Comment:
#if not defined(SIMU) && (defined(RADIO_FAMILY_T16) || defined(PCBNV14) || defined(RADIO_BOXER))